addons/mail/static/src/models/messaging_initializer/messaging_initializer.js

```diff
@@ -32,14 +32,18 @@ function factory(dependencies) {
                     model: 'mail.box',
                     name: this.env._t("Inbox"),
                 }]],
+                moderation: [['create', {
+                    id: 'moderation',
+                    model: 'mail.box',
+                    name: this.env._t("Moderation"),
+                }]],
                 starred: [['create', {
                     id: 'starred',
                     isServerPinned: true,
```

Comment:
It's weird to set "server pin" on mailboxes... Not to change in this PR, but something to have in mind for future improvement.

Reply:
This is just as weird as `_extendMessageDomain` on thread cache, and it comes from the same problem of using fake threads for those. Or from opposite angle, it comes from the server not storing inbox/moderation/etc in real channels for each user.